tests/core/typing/test_validation_typing.py

```diff
@@ -0,0 +1,221 @@
+"""Test static type annotations revealed by Mypy.
+
+This test will automatically analyze all files in the test validation_cases directory.
+To add new test cases, simply add a new .py file with each test case following
+the format:
+
+    reveal_type(arg)  # EXPECTED_TYPE: "<T>"
+
+where `arg` is any argument you want mypy to analyze, and <T> is the expected
+revealed type returned by mypy. Note: the output types from mypy are truncated
+with the module names removed, e.g. `typing.Sequence` -> `Sequence`,
+`builtins.float` -> `float`, etc.
+
+"""
```

Comment:
It took me a while, but I finally figured out how to create this great test. :)
I recommend adding a section to [Contributing](https://docs.pyvista.org/version/stable/extras/developer_notes) to understand how to add tests.

Reply:
This should be resolved with https://github.com/pyvista/pyvista/pull/5571/commits/88e4aff93a1c415a3f5cec9aa0d259cd4ec3ea07